lib/rules/no-array-prototype-extensions.js

```diff
@@ -89,6 +89,15 @@ const KNOWN_NON_ARRAY_FUNCTION_CALLS = new Set([
   'sessionStorage.clear()',
 ]);
 
+/**
+ * Ignore these function calls using RegExps.
+ */
+const KNOWN_NON_ARRAY_FUNCTION_CALLS_REGEXP = new Set([
+  // ember-cli-mirage
+  /(window|undefined)\.server\.schema\.(.*)\.findBy\(\)/,
```

Comment:
Why is `(window|undefined)` part of it? That doesn't seem related to this example.

Reply:
I can change the regexp to just `/\.server\.schema\.(.*)\.findBy\(\)/` if it seems more appropriate.

Reply:
Done, changed to simply `/\.server\.schema\.(.*)\.findBy\(\)/`